modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/toolbar/contributor/util/DLPortletToolbarContributorUtil.java

```diff
@@ -45,7 +41,7 @@ public Folder getFolder(
 			long folderId = ParamUtil.getLong(portletRequest, "folderId");
 
 			if (folderId != DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
-				folder = _dlAppLocalService.getFolder(folderId);
+				folder = DLAppLocalServiceUtil.getFolder(folderId);
```

Comment:
Hey @RenanUmV, using the `DLAppLocalServiceUtil` is not a good choice. Try to pass it as a method parameter instead. Thx